### DISCUSSION TITLE
hi

### DISCUSSION THREAD

**Discussion by abaganteng** [filamentphp/filament#13813] [General]
want learn

**Comment by dissto** [comment]
Hey,

here are some entry points to start your learning experience:

- [Filament docs](https://filamentphp.com/docs/3.x/panels/installation) (free)
- [Filament daily (youtube)](https://www.youtube.com/@FilamentDaily) (free)
- [Laravel daily](https://laraveldaily.com/tag/filament?source=homepage) (freemium)
- [Rapid Laravel Apps With Filament (laracasts)](https://laracasts.com/series/rapid-laravel-development-with-filament) (paid)
- [Build Advanced Components for Filament (laracasts)](https://laracasts.com/series/build-advanced-components-for-filament) (paid)
- [Filament support](https://filamentphp.com/docs/3.x/support/support) (free*)

> \* There is a paid option too
